obfx_modules/mystock-import/js/src/components/InsertImage.js

```diff
@@ -0,0 +1,13 @@
+const { createBlock } = wp.blocks;
+
+const InsertImage = (url = '', alt = '') => {
+	if(url === ''){
+		return false;
+	}
+	const block = createBlock("core/image", {
+		url: url,
+		alt: alt
+	});
+	wp.data.dispatch('core/editor').insertBlocks(block)
```

Comment:
Similarly, it will be:

```
const { dispatch } = wp.data;
const { insertBlocks } = dispatch( 'core/block-editor' ) || dispatch('core/editor');
```